src/components/plots/Header.jsx

```diff
@@ -8,52 +10,64 @@ import { useSelector, useDispatch } from 'react-redux';
 import { useRouter } from 'next/router';
 import PropTypes from 'prop-types';
 import moment from 'moment';
-import dynamic from 'next/dynamic';
 import { useBeforeunload } from 'react-beforeunload';
 import FeedbackButton from '../FeedbackButton';
 import { savePlotConfig } from '../../redux/actions/componentConfig/index';
 import itemRender from '../../utils/renderBreadcrumbLinks';
 import getFromApiExpectOK from '../../utils/getFromApiExpectOK';
-
 import { LOAD_CONFIG } from '../../redux/actionTypes/componentConfig';
 import { initialPlotConfigStates } from '../../redux/reducers/componentConfig/initialState';
 
-const KeyboardEventHandler = dynamic(
-  () => import('react-keyboard-event-handler'),
-  { ssr: false },
-);
-
 const Header = (props) => {
-  const { experimentId, plotUuid, finalRoute } = props;
+  const {
+    experimentId, plotUuid, finalRoute, testing,
+  } = props;
 
   const dispatch = useDispatch();
   const saved = !useSelector((state) => state.componentConfig[plotUuid]?.outstandingChanges);
   const lastUpdated = useSelector((state) => state.componentConfig[plotUuid]?.lastUpdated);
   const router = useRouter();
-  const type = useSelector((state) => state.componentConfig[plotUuid]?.type);
-  const { config, outstandingChanges } = useSelector((state) => state.componentConfig[plotUuid]) || {};
-  const reset = useRef(true);
+  const plotType = useSelector((state) => state.componentConfig[plotUuid]?.plotType);
```

Comment:
was that the bug? That we were calling `type` instead of `plotType`?

Reply:
Yes